packages/origin-ui-core/src/components/Modal/RoleChangedModal.tsx

```diff
@@ -159,6 +153,27 @@ export const RoleChangedModal = ({ showModal, setShowModal, setShowIRec }: IProp
                                     </ListItem>
                                 ))}
                             </List>
+                            {[Role.OrganizationAdmin].includes(user?.rights) && (
```

Comment:
There is a helper for this https://github.com/energywebfoundation/origin/blob/a20cd6458d564d22660b9f540437e3b5e01ff1c9/packages/origin-backend-core/src/User.ts#L47

Do `isRole(user, Role.OrganizationAdmin)` instead.